src/reviewRenderer/renderers/interactions/PortableCustomInteraction.js

```diff
@@ -22,5 +22,22 @@
  */
 import template from 'taoQtiItem/reviewRenderer/tpl/interactions/customInteraction';
 import portableCustomInteraction from 'taoQtiItem/qtiCommonRenderer/renderers/interactions/PortableCustomInteraction';
+import util from 'taoQtiItem/qtiItem/helper/util';
+import PortableElement from 'taoQtiItem/qtiCommonRenderer/helpers/PortableElement';
+import { isInteractionDisabled } from 'taoQtiItem/reviewRenderer/helpers/pci';
 
-export default Object.assign({}, portableCustomInteraction, {template});
\ No newline at end of file
+var getData = function getData(customInteraction, data) {
```

Comment:
es6 + side effects. Since the callback name sounds like a selector (get something), the callback shouldn't affect arguments. It should be called `updateDataAndReturn` in this case

Reply:
I can not change the name of the func because here in review renderer we are only overriding the functions present in the qtiCommonRenderer